Description: The parameters for some prototypes are defined as const so the functions 
 should use that too.
Author: Andreas Moog <amoog@ubuntu.com>
Bug-Debian: https://bugs.debian.org/701289
            https://bugs.debian.org/548384
Last-Update: 2013-08-17

Index: gltron/nebu/scripting/scripting.c
===================================================================
--- gltron.orig/nebu/scripting/scripting.c	2013-08-11 13:26:14.895792715 +0000
+++ gltron/nebu/scripting/scripting.c	2013-08-11 13:26:14.887792716 +0000
@@ -168,16 +168,16 @@
   return status;
 }    
 
-void scripting_RunFile(char *name) {
+void scripting_RunFile(const char *name) {
   lua_dofile(L, name);
 }
 
-void scripting_Run(char *command) {
+void scripting_Run(const char *command) {
   /* fprintf(stderr, "[command] %s\n", command); */
   lua_dostring(L, command);
 }
 
-void scripting_RunFormat(char *format, ... ) {
+void scripting_RunFormat(const char *format, ... ) {
   char buf[4096];
   va_list ap;
   va_start(ap, format);
Index: gltron/src/gltron.c
===================================================================
--- gltron.orig/src/gltron.c	2013-08-11 13:26:11.543792589 +0000
+++ gltron/src/gltron.c	2013-08-11 13:26:29.863793144 +0000
@@ -9,7 +9,7 @@
 #include "base/util.h"
 #include "SDL.h"
 
-int main(int argc, char *argv[] ) {
+int main(int argc, const char *argv[] ) {
 	initSubsystems(argc, argv);
 	runScript(PATH_SCRIPTS, "main.lua");
   return 0;
